ui/plugins/eu.esdihumboldt.hale.ui/src/eu/esdihumboldt/hale/ui/transformation/TransformDataWizardSourcePage.java

```diff
@@ -269,29 +312,9 @@ protected IOReport execute(final IOProvider provider, final IOReporter defaultRe
 				exportJob = new ExportJob(targetSink, writer, getAdvisor(),
 						DefaultReportHandler.getInstance());
 			}
-			else if (validationJob == null) {
```

Comment:
This seems to have become obsolete since the introduction of multiple validators.